config.yaml.example

```diff
@@ -70,14 +70,14 @@ storage:
   region: {{.STORAGE_REGION | default "us-east-1"}}
   endpoint: {{.STORAGE_ENDPOINT | default ""}}
 oauth:
-  base_URL: "{{.OAUTH_BASEURL | default "https://auth.denartcc.org" }}"
+  base_URL: "{{.OAUTH_BASEURL | default "https://auth.zdvartcc.org" }}"
```

Comment:
Do not change examples. This was not updated because I don't want working examples included.